system/jlib/jevent.hpp

```diff
@@ -21,6 +21,7 @@
 #include "jscm.hpp"
 #include "jatomic.hpp"
 #include "jbuff.hpp"
+#include "jlog.hpp"
```

Comment:
Not a big deal, but why include from the header when not used in the header?

Reply:
I will remove - it was probably a previous iteration that needed it.